### PR TITLE
Feature update with bim

### PR DESCRIPTION
**Pull request by GeorgFerdinandSchneider**
Added references to BIM and Protocol bindings as discussed with @HervePruvost  and @chung-edison 

Minor revision of text and ordering

Final PR before submitting to via a PR https://github.com/w3c/wot-usecases

### CODE REVIEW

File: USE-CASES/W3C-LBD-CG.md [hunk 1]
```diff
@@ -1,11 +1,8 @@
-# TODOs need to be removed!
-
-
-## Title: Smart Linked Building Data
+## Title: Smart Building Things
 
 ### Submitter(s):
 
-Edison Chung (MINES St. Etienne)
+[Edison Chung (MINES St. Etienne)](https://github.com/chung-edison)
```
Comment (chung-edison) [inline USE-CASES/W3C-LBD-CG.md]: https://orcid.org/0000-0002-5672-5508

File: USE-CASES/W3C-LBD-CG.md [hunk 2]
```diff
@@ -196,16 +88,14 @@ Another related case in smart buildings, which would greatly benefit from harmon
             "osh": "https://w3id.org/ibp/osh/OpenSmartHomeDataSet#",
             "bot": "https://w3id.org/bot#",
             "sosa": "http://www.w3.org/ns/sosa/",
```
Comment (chung-edison) [inline USE-CASES/W3C-LBD-CG.md:90]: should add `"brick": "https://brickschema.org/schema/1.1/Brick#",` to `@context`

File: USE-CASES/W3C-LBD-CG.md [hunk 3]
```diff
@@ -284,7 +159,7 @@ Example of a temperature setpoint actuator:
     ],
     "title": "Kitchen-tempS-Actuator",
     "description": "Kitchen Temperature Setpoint Actuator",
-    "@type": ["sosa:Actuator", "dog:ThermostaticRadiatorValve", "bot:element"],
+    "@type": ["sosa:Actuator", "brick:Zone_Air_Temperature_Setpoint", "bot:element"],
```
Comment (chung-edison) [inline USE-CASES/W3C-LBD-CG.md:163]: should add `"brick": "https://brickschema.org/schema/1.1/Brick#"` to `@context` and remove `"dog"`

**Review by chung-edison — CHANGES_REQUESTED** [review]
`"brick": "https://brickschema.org/schema/1.1/Brick#",` should be added to `@context`.

**Review by chung-edison — APPROVED** [review]

**Comment by gtfierro** [comment]
Brick is moving to a namespace without a version number, to simplify further development. The new Brick namespace, as of the [v1.2.0 release](https://github.com/BrickSchema/Brick/releases/tag/v1.2.0) is `https://brickschema.org/schema/Brick#`. We will be updating the website so that this URI redirects to the Turtle serialization of the file soon

**Comment by GeorgFerdinandSchneider** [comment]
Dear @gtfierro!

Thanks for pointing to the latest release of BRICK and the comment. We are happy to change the IRI of BRICK to the new one as soon as the redirects are in place.

Best

**Comment by gtfierro** [comment]
The redirect of [https://brickschema.org/schema/Brick#](https://brickschema.org/schema/Brick#) is now in place!

**Pull request merged**